core/src/saros/git/GitManager.java

```diff
@@ -0,0 +1,121 @@
+package saros.git;
+
+import java.io.File;
+import java.io.IOException;
+import org.apache.log4j.Logger;
+import org.picocontainer.Startable;
+import saros.activities.GitCollectActivity;
+import saros.activities.GitRequestActivity;
+import saros.activities.GitSendBundleActivity;
+import saros.annotations.Component;
+import saros.session.AbstractActivityConsumer;
+import saros.session.AbstractActivityProducer;
+import saros.session.IActivityConsumer;
+import saros.session.IActivityConsumer.Priority;
+import saros.session.internal.SarosSession;
+
+/**
+ * This Class is responsible for the negotiation between user that want to sent,receive and fetch
+ * commits.
+ *
+ * <p>In order to start the negotiation the user needs to call the procedure {@code
+ * sendCommitRequest()}.
+ *
+ * <p>Before starting negotiation the user need to set up his working directory tree by calling the
+ * procedure {@code changeWorkDirTree(File workDirTree)}. After calling it the first time the
+ * Manager hold a {@link JGitFacade} object.
+ */
+@Component(module = "core")
```

Comment:
The annotation is useless. We do not produce component diagrams since years.

Reply:
done